scripts/sonic-kdump-config

```diff
@@ -0,0 +1,633 @@
+        help='Enable kdump (Next image)')
+
+    # Disable kdump on Current Image
+    parser.add_argument('--disable', action='store_true',
+        help='Disable kdump')
+
+    # kdump status on Current Image
+    parser.add_argument('--status', action='store_true',
+        help='Show kdump status')
+
+    # Maximum number of kernel core dumps
+    parser.add_argument('--num_dumps', nargs='?', type=int, action='store', default=False,
+        help='Maximum number of kernel dump files stored')
+
+    # Memory allocated for capture kernel on Current Image
+    parser.add_argument('--memory', nargs='?', type=str, action='store', default=False,
+        help='Amount of memory reserved for the capture kernel')
+
+    # Capture kernel files
+    parser.add_argument('--files', action='store_true',
+        help='Show stored capture kernel files')
+
+    # Capture kernel file
+    parser.add_argument('--file', nargs=1, type=str,
+        help='Show stored capture kernel file')
+
+    # Show more information (used for sonic-kdump-config status)
+    parser.add_argument("-v", "--verbose", action='store_true',
+        help='displays detailed kdump status information. Used with status command.')
+
+    # Skips user from requiring to answer yes/no? to continue
+    parser.add_argument("-y", "--yes", action='store_true')
+
+    # Skips user from requiring to answer yes/no? to continue
+    parser.add_argument("-n", "--no", action='store_true')
+
+    # How many lines should we display from the kernel log
+    parser.add_argument("-l", "--lines", default=75, type=int,
+        help="Number of lines displayed from the kernel log")
+
+    # Validate input
+    if len(sys.argv[1:]) == 0:
+        parser.print_help()
+        sys.exit(1)
+
+    # Parse command arguments
+    options = parser.parse_args()
+
+    # Execute the command
+    changed = False
+    try:
+        if options.enable:
+            changed = cmd_kdump_enable(options.verbose, options.yes, options.no)
+        elif options.config_next:
+            changed = cmd_kdump_config_next(options.verbose, options.yes, options.no)
+        elif options.disable:
+            changed = cmd_kdump_disable(options.verbose, options.yes)
+        elif options.memory != False:
+            cmd_kdump_memory(options.verbose, options.memory)
+        elif options.num_dumps != False:
+            cmd_kdump_num_dumps(options.verbose, options.num_dumps)
+        elif options.status:
+            cmd_kdump_status()
+        elif options.files != False:
+            cmd_kdump_files()
+        elif options.file:
+            cmd_kdump_file(options.lines, options.file[0])
+        else:
+            parser.print_help()
+            sys.exit(1)
+    except Exception as e:
+        print_err('Error! Exception[%s] occured while processing the command sonic-kdump-config %s.' %(str(e), sys.argv[1]))
+        sys.exit(1)
+
+    if changed:
+        if options.yes:
+            perform_reboot()
```

Comment:
yes option is unclear to user. also user can choose to do warm reboot or fast reboot, recommend to do not integrate reboot into this config tool.